purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -194,6 +244,14 @@ data class PaywallData(
                 get() = listOfNotNull(header, background, icon)
         }
 
+        @Serializable
+        data class Tier(
+            val id: String,
+            val packages: List<String>,
+            @SerialName("default_package")
+            val defaultPackage: String,
```

Comment:
Nitpick I wonder if we should rename to `defaultPackageIdentifier`?

Reply:
I think this makes it clearer! Should I also rename the `defaultPackage` that is on `Configuration`? 🤔

Reply:
Ohhh right.... I mean, that would be a breaking change... we could probably deprecate it and add a new one, but I think it might be ok to keep it like this then... We can consider deprecating and renaming separately, and it would be good to be consistent.